apptest/testcase.go

```diff
@@ -15,17 +15,23 @@ type TestCase struct {
 	t   *testing.T
 	cli *Client
 
-	startedApps []Stopper
+	startedApps map[string]Stopper
 }
 
 // Stopper is an interface of objects that needs to be stopped via Stop() call
 type Stopper interface {
+	Name() string
```

Comment:
I needed something to use as a key in the map containing the set of started apps so that an app could be stopped later (see partial result test). This is why I introduced Name() - it is used as that key. I am open to other solutions that result in the same functionality.